src/scenes/ProgressReports/Detail/ProgressReportDrawer/ProgressReportStepper.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Box,
+  Paper,
+  Step,
+  StepButton,
+  StepConnector,
+  Stepper,
+  Theme,
+  Typography,
+} from '@mui/material';
+import { PeopleJoinedIcon } from '~/components/Icons';
+import { useProgressReportContext } from '../../ProgressReportContext';
+import { colorPalette } from './colorPalette';
+
+interface ProgressReportStepperProps {
+  step: number;
+}
+
+const stepperSx = (theme: Theme) => ({
+  '& .MuiStep-root .MuiStepLabel-label': {
+    fontSize: '0.75rem',
+  },
+  '& .MuiStep-root': {
```

Comment:
FWIW nesting works here too.
```suggestion
  '& .MuiStep-root': {
    '.MuiStepLabel-label' {
      fontSize: '0.75rem',
    }
```